### ISSUE TITLE
help file missing

### ISSUE THREAD

**Issue by koenr** [rwthmoodle/moodle-mod_cardbox#31]
on moodle/mod/cardbox/view.php?id=198&action=practice when in status

None of your cards are due for repetition yet.
If you practice anyway, correctly answered cards do not move on, but remain in their current tray.

After the word "yet" there is a question mark that should lead to "Help when no cards due"
There is no help displayed when clicking the question mark. 

I think the help exists in  help:nocardsdueforpractice | cardbox

**Comment by Amrita1991** [comment]
Thanks for reporting this. I have taken this in our Backlog

**Comment by RahelPhasen** [comment]
Unfortunantely we can't recreate this problem.
Is it still not working for you? Have you tried to use different browsers to see if that's maybe the problem?

**Comment by koenr** [comment]
Yes, you are right. It works for me too now (Firefox and Chrome).
No idea what went wrong :-(.

I'll close this issue as can't reproduce.
